spec/venue_spec.rb

```diff
@@ -43,13 +40,67 @@
   end
 
   describe '#yell_at_patrons' do
+    # Test that the return patron string values are converted to upper case.
     it 'returns a list of uppercased names' do
-      skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
+
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  describe '#over_capacity?' do
+    # Check if the patron list is under the capacity limit.
+    it 'checks at or below capacity limit' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+
+      expect(venue.over_capacity?).to eq(false)
+    end
+
+    # Check if the patron list is over the capacity limit.
+    it 'checks over capacity limit' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+
+      expect(venue.over_capacity?).to eq(true)
+    end
+  end
+
+  describe '#kick_out' do
+    # Test to verify that no one is kicked out when at or below capacity.
+    it 'it will not remove anyone if under capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.kick_out
+      
+      expect(venue.over_capacity?).to eq(false)
```

Comment:
If you are kicking out a patron, would your `venue.patrons` only then have `Mike and Megan`?